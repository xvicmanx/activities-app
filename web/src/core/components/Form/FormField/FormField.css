.FormField {
  margin: 0 0 15px;
  color: #767e84;
}

.FormField__label {
  display: block;
  margin: 0 0 10px;
}
