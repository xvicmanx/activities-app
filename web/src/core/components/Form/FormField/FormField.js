import React from 'react';
import './FormField.css';

const FormField = ({ label, children }) => {
  return (
    <div className="FormField">
      {label && <label className="FormField__label">{label}</label>}
      {children}
    </div>
  );
};

export default FormField;
