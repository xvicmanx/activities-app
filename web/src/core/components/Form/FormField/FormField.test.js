import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import FormField from './FormField';

describe('FormField', () => {
  it('renders properly', () => {
    const renderer = new ShallowRenderer();
    const onClick = jest.fn()
    const result = renderer.render(<FormField label="test">Field</FormField>);
    expect(result).toMatchSnapshot();
  });
});
