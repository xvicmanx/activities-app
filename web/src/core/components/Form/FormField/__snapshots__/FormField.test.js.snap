// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormField renders properly 1`] = `
<div
  className="FormField"
>
  <label
    className="FormField__label"
  >
    test
  </label>
  Field
</div>
`;
